src/plugins/discover/public/locator.ts

```diff
@@ -22,7 +22,7 @@ export interface DiscoverAppLocatorParams extends SerializableRecord {
   savedSearchId?: string;
 
   /**
-   * Optionally set index pattern ID.
+   * Optionally set index pattern / data view ID.
    */
   indexPatternId?: string;
```

Comment:
I assume we can't change this because apps outside Discover would need to change too?

Reply:
exactly!